sdk_container/src/third_party/coreos-overlay/app-crypt/clevis/metadata.xml

```diff
@@ -2,14 +2,15 @@
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
-		<email>julien@jroy.ca</email>
-		<name>Julien Roy</name>
+		<email>kjain7@u.rochester.edu</email>
+		<name>Krish Jain (based off Julien Roy's work) </name>
 	</maintainer>
 	<upstream>
 		<remote-id type="github">latchset/clevis</remote-id>
 	</upstream>
 	<use>
 		<flag name="luks">Enable LUKS support</flag>
 		<flag name="tpm">Enable TPM support</flag>
+		<flag name="dracut">Enable dracut support</flag>
```

Comment:
I don't think we need a new use flag here but can directly depend on dracut

Reply:
Sounds good. I removed the use flag.